docs/source/database.rst

```diff
@@ -0,0 +1,20 @@
+Database
+========
+
+Database Migration
+------------------
+
+For database migration we are using `Alembic <https://alembic.sqlalchemy.org/en/latest/>`_.
+If you have a database that is already on the latest database schema, just execute
```

Comment:
```suggestion
To add the alembic metadata to the database and make it ready for later migration, run:
```

Or did I misunderstand this and the `emissionsapi` code must match the currently deployed database schema?

Reply:
`alembic stamp head` writes the current version into a meta table in the database. So if you setup a database with the current schema, you can run this command so that alembic knows the version of your database.

If there is then a database migration in the future, alembic would start by this version.